src/test/mir-opt/tls-access.rs

```diff
@@ -11,3 +11,4 @@ fn main() {
 }
 
 // EMIT_MIR tls_access.main.SimplifyCfg-final.after.mir
+// compile-flags: -Zmir-opt-level=0
```

Comment:
Compiletest annotations [are only parsed until the first `mod` or `fn`](https://github.com/rust-lang/rust/blob/f001f9301c889101d8a71358b64b96e9707c832b/src/tools/compiletest/src/header.rs#L521-L527), so this directive is ignored, which is likely not what you wanted to do.

I discovered this because I'm running the test suite on a private fork that parses compiletest annotations for the whole file (see https://github.com/rust-lang/compiler-team/issues/512), and this resulted in a test failure:

```
thread '[mir-opt] src/test/mir-opt/tls-access.rs' panicked at 'Output file `/foo/build/x86_64-unknown-linux-gnu/test/mir-opt/tls-access/tls_access.main.SimplifyCfg-final.after.mir` from test does not exist, available files are in `/foo/build/x86_64-unknown-linux-gnu/test/mir-opt/tls-access`', src/tools/compiletest/src/runtest.rs:3396:25
```